Containerfile

```diff
@@ -119,7 +118,68 @@ RUN make build-daemon
+pushd /build/windows-machine-config-operator/kubelet/_output/local/bin/windows/amd64/ && tar -cf - kube-log-runner.exe | gzip -9 > /payload/kube-node/kube-log-runner.exe.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/kubelet/_output/local/bin/windows/amd64/kube-log-runner.exe >> sha256sum && \
+pushd /build/windows-machine-config-operator/kubelet/_output/local/bin/windows/amd64/ && tar -cf - kube-proxy.exe | gzip -9 > /payload/kube-node/kube-proxy.exe.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/kubelet/_output/local/bin/windows/amd64/kube-proxy.exe >> sha256sum && \
+# Copy CNI plugin binaries
+mkdir /payload/cni && \
+pushd /build/windows-machine-config-operator/containernetworking-plugins/bin/ && tar -cf - host-local.exe | gzip -9 > /payload/cni/host-local.exe.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/containernetworking-plugins/bin/host-local.exe >> sha256sum && \
+pushd /build/windows-machine-config-operator/containernetworking-plugins/bin/ && tar -cf - win-bridge.exe | gzip -9 > /payload/cni/win-bridge.exe.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/containernetworking-plugins/bin/win-bridge.exe >> sha256sum && \
+pushd /build/windows-machine-config-operator/containernetworking-plugins/bin/ && tar -cf - win-overlay.exe | gzip -9 > /payload/cni/win-overlay.exe.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/containernetworking-plugins/bin/win-overlay.exe >> sha256sum && \
+# Build csi-proxy.exe
+mkdir /payload/csi-proxy && \
+pushd /build/windows-machine-config-operator/csi-proxy/bin/ && tar -cf - csi-proxy.exe | gzip -9 > /payload/csi-proxy/csi-proxy.exe.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/csi-proxy/bin/csi-proxy.exe >> sha256sum && \
+# Copy required powershell scripts
+mkdir /payload/powershell && \
+pushd /build/windows-machine-config-operator/pkg/internal/ && tar -cf - gcp-get-hostname.ps1 | gzip -9 > /payload/powershell/gcp-get-hostname.ps1.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/pkg/internal/gcp-get-hostname.ps1 >> sha256sum && \
+pushd /build/windows-machine-config-operator/pkg/internal/ && tar -cf - windows-defender-exclusion.ps1 | gzip -9 > /payload/powershell/windows-defender-exclusion.ps1.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/pkg/internal/windows-defender-exclusion.ps1 >> sha256sum && \
+pushd /build/windows-machine-config-operator/pkg/internal/ && tar -cf - hns.psm1 | gzip -9 > /payload/powershell/hns.psm1.tar.gz && popd && \
+sha256sum /build/windows-machine-config-operator/pkg/internal/hns.psm1 >> sha256sum && \
+chmod -R 644 /payload && chmod -R +X /payload &&\
+# Create directory for generated files with open permissions, this allows WMCO to write to this directory
+mkdir /payload/generated && \
+chmod 0777 /payload/generated
```

Comment:
for my own education:

why is Read + Write + Execute needed for group and others?